WcaOnRails/app/models/competition.rb

```diff
@@ -83,6 +83,15 @@ class Competition < ApplicationRecord
     currency_code
     enable_donations
     registration_requirements
+    free_competition_enabled
+    on_the_spot_registration_enabled
+    on_the_spot_entry_fee
+    on_the_spot_entry_fee_lowest_denomination
+    refund_policy
+    refund_policy_amount
+    guests_entry_fee_enabled
```

Comment:
As we discussed offline, I don't think any of these *_enabled fields need to make it to the backend. I think they should just be frontend checkboxes that determine if certain other fields are visible.